src/store/metadata/stub/content/editions/persistent-absences-for-pupils_2023-05.json

```diff
@@ -0,0 +1,107 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/persistent-absences-for-pupils/editions",
+    "@type": "hydra:Collection",
+    "title": "Persistent absences for pupils looked after by local authorities",
+    "editions": [
+      {
+        "@id": "https://staging.idpd.uk/datasets/persistent-absences-for-pupils/editions/2023-05",
+        "@type": "dcat:Dataset",
+        "in_series": "https://staging.idpd.uk/datasets/persistent-absences-for-pupils",
+        "identifier": "2023-05",
+        "title": "Persistent absences for pupils looked after by local authorities",
+        "summary": "Percentage of pupils who have been looked after by the local authorities over financial years 2016/17 - 2021/22.",
+        "description": "This dataset shows the percentage of pupils in state-funded primary, state funded secondary, special schools and pupil referral units who have been looked after continuously for at least 12 months (as of 31 March of that year) by local authorities who were persistently absent (those absent for 10% of schooling sessions) during the academic year, in England for financial years 2016/17 - 2021/22.",
+        "publisher": "https://staging.idpd.uk/publishers/department-for-education",
+        "creator": "https://staging.idpd.uk/publishers/department-for-education",
+        "contact_point": {
+          "name": "ofsted",
+          "email": "mailto:informationrequest@ofsted.gov.uk"
+        },
+        "topics": [
+          "https://staging.idpd.uk/topics/children-education-and-skills"],
+        "frequency": "annual",
+        "keywords": [
+          "state-funded",
+          "children-out-of-school",
+          "schools",
+          "local-authority",
+          "subnational",
+          "county",
+          "region"
+        ],
+        "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+        "issued": "2023-05-16T13:35:25Z",
+        "modified": "2023-09-01T12:30:00Z",
+        "spatial_resolution":["E06", "E07", "E08", "E09", "E11"],
+        "spatial_coverage": "E92000001",
+        "temporal_resolution": ["P1Y"],
+        "temporal_coverage": {
+          "start": "2022-01-01T12:00:00Z",
+          "end": "2022-12-31T12:00:00Z"
+        },
+        "versions_url": "https://staging.idpd.uk/datasets/persistent-absences-for-pupils/editions/2023-05/versions",
```

Comment:
I think you should use the full dataset name as there is another dataset - `persistent-absences-for-all-pupils`, to avoid any confusion